cdap-data-fabric/src/main/java/io/cdap/cdap/spi/metadata/dataset/SearchHelper.java

```diff
@@ -332,6 +375,34 @@ private MetadataSearchResponse search(Set<MetadataScope> scopes, SearchRequest r
     return result;
   }
 
+//  private Set<MetadataEntity> getSortedEntities(List<MetadataEntry> results, SortInfo sortInfo) {
```

Comment:
lets keep the original method and modify it instead of commenting it.

Reply:
Addressed in [#11489](https://github.com/cdapio/cdap/pull/11489)